### DISCUSSION TITLE
Hi 👋

### DISCUSSION THREAD

**Discussion by Marciox001** [Ileriayo/markdown-badges#390] [Ideas]
I'm interested

**Comment by Abhishek10351** [comment]
Hello

**Comment by Marciox001** [comment]
How are you?

**Comment by TheSeer507** [comment]
Hello @Ileriayo 
Didn't know where to put this, just wanted to say thank you. This is an amazing work that surely will come handy in making any README.md much more appealing. Thanks again and I will try to support your work and contribute as well.
Cheers!

**Reply by Ileriayo** [reply]
Hey @TheSeer507,

Thanks for your kind words!
